src/components/partner/Partner.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+<div>
+  <h2>Partner:</h2>
+    <v-carousel
+    cycle
+    height="200"
```

Comment:
Is it possible to make the size of the logos smaller? On desktop they are already very large